src/utils/emasser/generateConfig.ts

```diff
@@ -0,0 +1,274 @@
+import path from 'path'
+import colors from 'colors' // eslint-disable-line no-restricted-imports
+import fse from 'fs-extra'
+import dotenv from 'dotenv'
+import inquirer from 'inquirer'
+import inquirerFileTreeSelection from 'inquirer-file-tree-selection-prompt'
+
+const PROMPT_MESSAGE = [
+  'Provide the eMASS API key (api-key) - valid key is > 30 alpha numeric characters:',
+  'Provide the eMASS User unique identifier (user-uid):',
+  'Provide the FQDN for the eMASS server:',
+  'Provide the eMASS key.pem private encrypting the key in PEM format (file, include the path):',
+  'Provide the eMASS cert.pem containing the certificate information in PEM format (file, include the path):',
+  'Provide the password for the private encryption key.pem file (no validation is performed):',
+  'Provide the server communication port number (default is 443):',
+  'Server requests a certificate from clients - true or false (default true):',
+  'Reject connection not authorized with the list of supplied CAs- true or false (default true):',
+  'Set debugging on or off - true or false (default false):',
+  'Display null value fields - true or false (default true):',
+  'Convert epoch to data/time value - true or false (default false):',
+]
```

Comment:
Add extra prompt at the end that tells users to do `emasser get test_connection` and `emasser get -h` for more options.